index.html

```diff
@@ -182,15 +182,15 @@ <h3>
       provider=].
     </p>
     <h2>
-      The `IdentityRequestProvider` dictionary
+      The `IdentityRequestDetails` dictionary
```

Comment:
Can we call this `DigitalIdentityRequest` as opposed to use the `Details` suffix? `Details` seems a bit off to me (although, surprisingly, there is a lot of "Details" in chrome, so maybe that's just me). Anyway, I think it is likely that whatever name you pick for this dictionary is likely to be temporary, because we need to form an opinion of what a `providers` is, so maybe we can just rename this when we figure out what to call `providers`?

[1] https://source.chromium.org/search?q=file:idl%20Details%20-file:out&sq=&ss=chromium

Reply:
yeah, that might be better indeed.